ecosystem/model.py

```diff
@@ -86,21 +87,18 @@ def count_type(model, patch_condition):
                 count += 1
         return count
 
-
-    @staticmethod
-    def calculate_local_densities(model):
+    def calculate_local_densities(self, model):
         '''Helper method to count vegetated neighbours.'''
 
         qplusplus = []
-        qplusminus = []
+        qminusplus = []
```

Comment:
It is a bit confusing since in the figures there is still `qplusminus`, but here it is replaced. Which one is it?